middleware/packet-forward-middleware/e2e/setup.go

```diff
@@ -49,12 +49,12 @@ var (
 		TrustingPeriod: "336h",
 		NoHostMount:    false,
 		EncodingConfig: encoding(),
-		Env:            []string{"NON_REFUNDABLE_TEST=true"},
+		// Env:            []string{"NON_REFUNDABLE_TEST=true"},
```

Comment:
ref: https://github.com/strangelove-ventures/interchaintest/pull/933 will add this functionality back to v8

Reply:
using this commit, solves the issue